docs/configuration/options.md

```diff
@@ -155,12 +155,12 @@ Interval for evicting outdated project configs from memory.
 
 ## Size Limits
 
-Controls various HTTP-related limits.  All values are either integers or are human-readable strings of a number
-and a human-readable unit, such as:
+Controls various HTTP-related limits. All values are either integers or are
+human-readable strings of a number and a human-readable unit, such as:
 
 - `500B`
-- `1KB` (*1,000* bytes)
-- `1KiB` (*1,024* bytes)
+- `1kB` (*1,000* bytes)
+- `1KB` or `1KiB` (*1,024* bytes)
```

Comment:
Does that mean `1KB` is an alias for `1KiB`, but `1MB` isn't?

Reply:
Apparently that's how `human_size` implements it 🤷